### DISCUSSION TITLE
Παρουσίαση ενδιάμεσης προόδου και Αναπλήρωση

### DISCUSSION THREAD

**Discussion by epidrome** [upatras-hci/iv#55] [Announcements]
Για τις επόμενες συναντήσεις μέχρι τις γιορτές θα γίνουν οι παρουσιάσεις ενδιάμεσης προόδου, δλδ τουλάχιστον μέχρι και το παραδοτέο 6.

Για τον σκοπό αυτό, απαντήστε παρακάτω ποια εβδομάδα θέλετε να παρουσιάσετε την πρόοδο σας, ώστε μετά να συμπληρωθεί ο πίνακας:

| Εβδομάδα | Όνομα |
| -- | -- |
| 24 Νοε | |
| 1 Δεκ | [1](https://github.com/e-liouta) |
| 8 Δεκ  | [1](https://github.com/akimo13) |
| 15 Δεκ. | [1](https://github.com/MairySs) |
| 22 Δεκ. | [1](https://github.com/VasilisK23) [2](https://github.com/EvangeliaChatz) |

Ο σκοπός της παρουσίασης είναι να δείξετε την αναφορά σας και να γίνουν σχόλια βελτίωσης. Σε κάθε συνάντηση μπορούμε να έχουμε μόνο 1-2 παρουσιάσεις έτσι ώστε να υπάρχει χρόνος για να δούμε τις τρέχουσες δραστηριότητες και την θεωρία.

Η σημερινή συνάντηση 17 Νοε θα αναπληρωθεί μετά τις γιορτές και κατά την περίοδο των εξετάσεων αφού το μάθημα αυτό δεν έχει τελική γραπτή εξέταση

**Comment by e-liouta** [comment]
Καλησπέρα σας! Θα ήθελα να παρουσιάσω 1 Δεκ. 
Ενδεχομένως να έχω λύσει κάποια θέματα που έχω και μέχρι τις 24 Νοε. οπότε αν δεν έχει γεμίσει η ημερομηνία με παρουσιάσεις συναδέλφων ίσως παρουσιάσω και τότε.

**Comment by akimo13** [comment]
Γειά σας! Θα ήθελα να παρουσιάσω στις 8 Δεκεμβρίου, πιστεύω πως μέχρι τότε θα έχω αντιμετωπίσει επιτυχώς τα θέματα που έχουν παρουσιαστεί. Ευχαριστώ!

**Comment by MairySs** [comment]
Καλησπέρα, θα ήθελα να παρουσιάσω 15 Δεκεμβρίου.

**Comment by VasilisK23** [comment]
Χαιρεται, αν δεν υπαρχει αντιρρηση, θα ηθελα να παρουσιασω στις 22 Δεκεμβριου. Ευχαριστω!

**Comment by EvangeliaChatz** [comment]
Θα παρουσιάσω κι εγώ στις 22 Δεκεμβρίου!
